web/html/tables/caption.md

```diff
@@ -52,7 +55,9 @@ Example:
 </table>
 ```
 
-> 💡 In the example above, the `<strong>` element is used to bring importance to the "Name" and "Starting Date" tags. However, it can be replaced with the `style="font-weight:bold;"` CSS property. 
+> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 
+
+If you'd like to only style the elements bold without bringing attention to them, you should replace it with the `style="font-weight:bold;"` CSS property.
```

Comment:
What is "it" in this case?

Reply:
the `<b>` element.

These two are connected: 

> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 

If you'd like to only style the elements bold without bringing attention to them, you should replace it with the `style="font-weight:bold;"` CSS property.



Should I add it as a single note?

Reply:
I've added a span and modified the wording, lmk if it's good